src/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { createRoot } from 'react-dom/client';
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import { getMission } from './redux/mission';
+import { fetchRockets } from './redux/rockets';
+import App from './App';
+import './styles/index.css';
+
+const container = document.getElementById('root');
+const root = createRoot(container);
+
+store.dispatch(getMission());
```

Comment:
- [ ] Your mission action should not be dispatched when the app loads. It should only be dispatched when the user navigates to the Mission page. Fix this by removing this mission dispatch here. The one in the Mission component will work just fine. Only the rocket action should be triggered on app load.